website/content/en/docs/user-guide.md

```diff
@@ -249,6 +249,89 @@ return reconcile.Result{RequeueAfter: time.Second*5}, nil
 
 **Note:** Returning `Result` with `RequeueAfter` set is how you can periodically reconcile a CR.
 
```

Comment:
Apply the changes in the doc here as well.

Reply:
Done

Reply:
Done! Thanks for the great suggestions!